lib/miio/Model.js

```diff
@@ -1,17 +1,20 @@
-/**
- * @param {string} name
- * @constructor
- */
-const Model = function(name) {
-    this.viomiApi = name == "viomi.vacuum.v7";
-    this.name = name;
-};
+class Model {
+    /**
+     * @param {string} name
+     * @constructor
+     */
+    constructor(name) {
+        this.viomiApi = name == "viomi.vacuum.v7";
+        this.RoborockApi = name == "roborock.vacuum.s5";
```

Comment:
This works for what you use it for, but doesn't really indicate what the naming implies. The naming implies that for a Roborock vacuum the property `RoborockApi` is true, while for a Viomi vacuum the property `viomiApi` is true. Instead the property `viomiApi` is true for all currently supported Viomi vacuums and false for all currently supported Roborock vacuums. The `RoborockApi` property does not properly indicate that the `Roborock` vacuum API should be used, but that the robot is a Gen 2 Roborock vacuum and supports persistent data. 

Thinking about future supported devices, maybe we shouldn't set booleans here to define what implementation of `MiioVacuum` to use, but rather set a single property that already references the correct implementation to use.

Reply:
This should be refactored later, it's not the topic of this pull request.

Reply:
Fair enough.